EmotiBit.cpp

```diff
@@ -1125,9 +1126,7 @@ bool EmotiBit::setupSdCard()
 	if (testingMode == TestingMode::ACUTE || testingMode == TestingMode::CHRONIC)
 	{
 		// list all files on SD-Card
-#ifdef ADAFRUIT_FEATHER_M0
-		SD.ls(LS_R);
-#elif defined ARDUINO_FEATHER_ESP32
+#if defined ARDUINO_FEATHER_ESP32
```

Comment:
conform to `#ifdef ARDUINO_FEATHER_ESP32`
That seems to be the pattern we are following.